### PR TITLE
removed "resolveActivity" check on open camera

### PR DESCRIPTION
**Pull request by tobiasjungmann**
## Issue
This PR is a fix for a small bug I stumbled on. Since I found a solution rather quickly, I've skipped the creating an additional issue.

The problem was that the camera view in the feedback activity could not be opened.

## Solution
This code (FeedbackPresenter) finished the method too early:

```
if (takePictureIntent.resolveActivity(context.packageManager) == null) {  
    return  
}
```

This check returns null even if successful in more recent API versions. This check is now removed and the potential exception will be caught if the activity does actually not exist. 

## How to test
Open Feedback tab and add a new image with the camera. Camera view should open.

### CODE REVIEW

**Comment by CommanderStorm** [comment]
I will review this once my exams are done next week.

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/feedback/FeedbackPresenter.kt [hunk 1]
```diff
@@ -127,8 +125,11 @@ class FeedbackPresenter @Inject constructor(
         val authority = "de.tum.in.tumcampusapp.fileprovider"
         val photoURI = FileProvider.getUriForFile(context, authority, photoFile)
         takePictureIntent.putExtra(MediaStore.EXTRA_OUTPUT, photoURI)
-
-        view?.openCamera(takePictureIntent)
+        try {
+            view?.openCamera(takePictureIntent)
+        } catch (e: ActivityNotFoundException) {
+            photoFile.delete()
```
Comment (Bentipa) [inline app/src/main/java/de/tum/in/tumcampusapp/component/tumui/feedback/FeedbackPresenter.kt:131]: why is this required? is the file already created when getting the Uri?

Comment (tobiasjungmann) [reply app/src/main/java/de/tum/in/tumcampusapp/component/tumui/feedback/FeedbackPresenter.kt:131]: The file is created before trying to open the new activity (line 115). If the activity cannot be opened, the file won't be used. Therefore, it should be deleted again. Previously, the file would not even be created if the check which I have removed failed.

**Review by CommanderStorm — APPROVED** [review]

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/feedback/FeedbackPresenter.kt (hunk 1 above)
Comment (Bentipa) [reply app/src/main/java/de/tum/in/tumcampusapp/component/tumui/feedback/FeedbackPresenter.kt:131]: Ah sorry, did not open the collapsed section :D
